packages/interactivity/src/directives.tsx

```diff
@@ -1,8 +1,6 @@
 // eslint-disable-next-line eslint-comments/disable-enable-pair
 /* eslint-disable react-hooks/exhaustive-deps */
 
-/* @jsx createElement */
```

Comment:
This doesn't seem to work anymore and the importSource annotation neither. I think since there are very few JSX elements, it's ok to avoid JSX in these files.

Reply:
@sirreal, it looks like JSX is no longer necessary with the refactoring included in `@wordpress/interactivity`. It would be great to have a quick check from you to confirm everything is still looking good.

Reply:
I was going to flag this, but @youknowriad fixed it. I'm happy to remove the JSX like you did here 👍 

Optionally, the `createElement` alias could be removed now (just use `h`).

This defines the directives, so either Interactivity API is probably completely broken or working fine. I'll test and report back.

Reply:
I tested the navigation block with client side navigation in the playground and it worked as expected 👍